docs/src/main/asciidoc/extension-codestart.adoc

```diff
@@ -69,6 +69,10 @@ Each codestart consists of:
 
 The https://github.com/quarkusio/quarkus/tree/main/independent-projects/tools/base-codestarts/src/main/resources/codestarts/quarkus[base codestarts, window="_blank"] contains templates to create project, buildtool, languages, config and tooling files.
 
+Quarkus also provides the <<building-my-first-extension.adoc#maven-setup,`create-extension` Maven Mojo>> to initialize an extension project.
+
+Adding the `--with-codestart` option to this Mojo generates a Codestart skeleton for the extension.
```

Comment:
Is that the expected option when executing the Mojo or is that from the Quarkus CLI? I thought Maven would understand `-D` flags instead.

Reply:
Also, adding an example would help

Reply:
Yes, you are correct @gastaldi. Was doing things in a hurry so I messed up.

Should be corrected now and added the example as suggested by you.